api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -270,6 +270,16 @@ public void createAllOfUsBillingProject(String projectName) {
         });
   }
 
+  @Override
+  public void deleteBillingProject(String billingProject) {
+    BillingApi billingApi = billingApiProvider.get();
+    retryHandler.run(
+        (context) -> {
+          billingApi.deleteBillingProject(billingProject);
```

Comment:
Does this immediately expunge the GCP project, or does it get put into a queue for deletion? There's no wrong answer, as long as everyone understands the behavior.

Reply:
Also, is there a ticket for a simple cron to find orphaned billing projects?

Reply:
Immediate deletion.

We currently have a "Garbage Collection" cron for orphaned billing projects, but we should revisit this process.  I have a plan to do this, but it's not in JIRA yet